src/java/main/org/apache/zookeeper/ZooDefs.java

```diff
@@ -69,6 +69,8 @@
 
         public final int deleteContainer = 20;
 
+        public final int getChildrenPaginated = 71;
```

Comment:
Instead of random number, I'd prefer to use next sequence getChildrenPaginated = 21;